frontend/src/metabase/query_builder/components/QuestionActions.tsx

```diff
@@ -70,6 +73,23 @@ const QuestionActions = ({
   const isMetabotEnabled = useSelector(state =>
     getSetting(state, "is-metabot-enabled"),
   );
+
+  const canUpload = useSelector(state => {
+    const uploadsEnabled = getSetting(state, "uploads-enabled");
```

Comment:
Do we want to move this to `query_builder/selectors`? Doesn't really change much, but generally selector logic isn't so directly in components